### PR TITLE
chore: label correspondingly development nodes

### PR DESCRIPTION
**Pull request by flacatus**
- Make sure all development cluster will have proper nodes labels

### CODE REVIEW

**Review by mkosiarc — APPROVED** [review]

**Comment by manish-jangra** [comment]
looks good to me

File: hack/preview.sh [hunk 1]
```diff
@@ -141,6 +141,24 @@ if [ "$EC_DISABLE_DOWNLOAD_SERVICE" = "true" ]; then
...
+for node in $(kubectl get nodes -o name); do
+    echo "labeling $node..."
+    if kubectl label $node konflux-ci.dev/workload=konflux-tenants --overwrite; then
+        echo "successfully labeled $node"
+    else
+        echo "failed to label $node"
+    fi
+done
+
+kubectl get nodes --show-labels | grep konflux-ci.dev/workload
```
Comment (enarha) [inline hack/preview.sh:154]: I think that the return code will have a 0 value even if one match (labeled node) is found. What you can do instead is take the count of the nodes and compare to the count of matched lines.

Comment (enarha) [reply hack/preview.sh:154]: ... or do a loop on the nodes and check for each one. I like the above option more.

Comment (flacatus) [reply hack/preview.sh]: i added a count for nodes.

**Review by kasemAlem — APPROVED** [review]

**Pull request merged**